databricks-sdk-java/src/main/java/com/databricks/sdk/core/commons/CommonsHttpClient.java

```diff
@@ -41,6 +44,23 @@ public CommonsHttpClient(int timeoutSeconds) {
     hc = makeClosableHttpClient();
   }
 
+  public CommonsHttpClient(DatabricksConfig databricksConfig) {
+    int timeoutSeconds =
+        databricksConfig.getHttpTimeoutSeconds() == null
+            ? 300
+            : databricksConfig.getHttpTimeoutSeconds();
+    timeout = timeoutSeconds * 1000;
+    connectionManager.setMaxTotal(100);
+    ProxyConfig proxyConfig = new ProxyConfig(databricksConfig);
+    hc = makeClosableHttpClient(proxyConfig);
```

Comment:
Can we condense the logic into a single constructor?
```suggestion
    this(databricksConfig.getHttpTimeoutSeconds() == null
            ? 300
            : databricksConfig.getHttpTimeoutSeconds(),
            new ProxyConfig(databricksConfig));
```